lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -2143,6 +2143,25 @@ NvidiaMmaEncodingAttr::getSizePerThreadForOperands(unsigned opIdx) const {
   }
 }
 
+// Temporary solution for fetching the number of elements processed by a thread
+// in a single pseudo-tile during an MFMA dot operation. "Pseudo-tile" refers to
+// a conceptual grouping rather than an actual hardware tile, defined by the
+// kWidth parameter.
+// This function is a temporary workaround because getSizePerThreadForOperands
+// does not currently consider the kWidth parameter (an update is in progress to
```

Comment:
@lezcano I think it will also be supported by your ongoing work?

Reply:
Yes, I was actually referring to your work in this comment. However, I was told that it won't land soon so I implemented this workaround to be able to have vectorized loads in the meantime. Feel free to remove this code in your PR.